documentation/airport-format.md

```diff
@@ -1,179 +1,168 @@
-    "app": "controller callsign for approach control",
-    "dep": "controller callsign for departure control"
+    "twr": "tower callsign",                // The callsign of the airport control tower.
+    "app": "approach control callsign",     // The callsign of the airport approach tower.
+    "dep": "departure control callsign"     // The callsign of the airport departure tower.
   },
-  "icao": "KSFO",             // uppercase ICAO airport code
-  "iata": "SFO",              // uppercase IATA airport code
-  "magnetic_north": 13.7,     // magnetic declination, in degrees EAST!
-  "ctr_radius": 80,           // radius from 'position' that the airspace extends
-  "ctr_ceiling": 10000,       // elevation up to which the airspace extends
-  "initial_alt": 5000         // alt departures climb to if given "as-filed" clearance, but no "climb-via-sid" or altitude assignment
-  "position", ["lat", "lon", "elevation"]  // the latitude/longitude of the "center" of the airport; see comments below
-  "rr_radius_nm": 5,          // radius of range rings, nautical miles
-  "rr_center": ["lat", "lon"],// position where range rings are centered, nautical miles
-  "has_terrain": true,        // true/false for if has an associated GeoJSON terrain file in assets/airports/terrain
-  "wind": {     // wind is used for score, and can affect aircraft's ground tracks if enabled in settings
-    "angle": 0, // the heading, in degrees, that the wind is coming from
-    "speed": 3  // the speed, in knots, of the wind
+  "icao": "KSFO",                           // Uppercase ICAO airport code.
+  "iata": "SFO",                            // Uppercase IATA airport code.
+  "magnetic_north": 13.7,                   // Magnetic deviation from Magnetic North, in degrees East. (can be a negative number)
+  "ctr_radius": 80,                         // Radius of the airspace from the center of the airport. (in nautical miles)
+  "ctr_ceiling": 10000,                     // Height of the airspace. (in feet)
+  "initial_alt": 5000,                      // Set altitude for departures to climb if given "clearance as filed", but without a "climb via SID" clearance or altitude assignment.  
+  "position": ["lat", "lon", "elevation"],  // The center of the airport. (in latitude, longitude, and elevation) // See comments below.
+  "rr_radius_nm": 5,                        // Radius of the ring sections. (in nautical miles)
+  "rr_center": ["lat", "lon"],              // The center position of the rings. (in nautical miles)
+  "has_terrain": true,                      // Whether or not the airport has an associated GeoJSON terrain file in "assets/airports/terrain". (true or false)
+  "wind": {                                 // Wind is used for score. // It can affect an aircraft's ground tracks if enabled in settings.
+    "angle": 0,                             // Wind direction. (in degrees)
+    "speed": 3                              // Wind speed. (in knots)
   },
-  "fixes": {
```

Comment:
Add that fixes should ideally be organised in alphabetical order for readability.